$OpenBSD: patch-src_SSL_cpp,v 1.1 2020/03/21 16:19:27 kn Exp $

Prepend "TLSv1.3" to the default cipher string until libssl can cope.

Index: src/SSL.cpp
--- src/SSL.cpp.orig
+++ src/SSL.cpp
@@ -39,7 +39,7 @@ void MumbleSSL::destroy() {
 }
 
 QString MumbleSSL::defaultOpenSSLCipherString() {
-	return QLatin1String("EECDH+AESGCM:EDH+aRSA+AESGCM:DHE-RSA-AES256-SHA:DHE-RSA-AES128-SHA:AES256-SHA:AES128-SHA");
+	return QLatin1String("TLSv1.3:EECDH+AESGCM:EDH+aRSA+AESGCM:DHE-RSA-AES256-SHA:DHE-RSA-AES128-SHA:AES256-SHA:AES128-SHA");
 }
 
 QList<QSslCipher> MumbleSSL::ciphersFromOpenSSLCipherString(QString cipherString) {
